apm-agent-core/src/main/java/co/elastic/apm/agent/report/serialize/DslJsonSerializer.java

```diff
@@ -658,7 +658,7 @@ private void serializeSpanContext(SpanContext context) {
                 jw.writeByte(COMMA);
             }
             writeFieldName("tags");
```

Comment:
Should we mark such places for a reminder to change on 2.0?

Reply:
There is currently no concrete plan when the intake API will be changed.